docker-compose.yaml

```diff
@@ -0,0 +1,39 @@
+#https://docs.docker.com/registry/deploying/#deploy-your-registry-using-a-compose-file
+services:
+  nginx:
+    image: "nginx:alpine"
```

Comment:
Let's use `nginx:stable-alpine`.  Boring and safe vs cutting edge!

Reply:
Do we need to use bcrypt?

![image](https://user-images.githubusercontent.com/59970326/155909608-356c6791-ba56-4928-bb07-f27d5e61f472.png)

Reply:
`nginx:stable-alpine` is still alpine, just a stable vs. development version of nginx